proto/iotics/api/input.proto

```diff
@@ -30,6 +31,12 @@ service InputAPI {
 
   // Receives input messages for a specific input.
   rpc ReceiveInputMessages(ReceiveInputMessageRequest) returns (stream ReceiveInputMessageResponse) {}
+
+  // Creates an input. (local and remote)
```

Comment:
- Idempotent, local only (see e.g. `CreateFeed` rpc)
- I think it would be worth mirroring the doc strings of the corresponding methods in `feed.proto`